Dockerfile

```diff
@@ -62,5 +62,5 @@ RUN virtualenv --no-site-packages -p python2 /opt/letsencrypt/venv && \
 # bash" and investigate, apply patches, etc.
 
 ENV PATH /opt/letsencrypt/venv/bin:$PATH
-# TODO: is --text really necessary?
```

Comment:
That's a better solution if it works reliably. Tested on Arch and `dialog` worked fine in Docker.